File: maxhanna.Server/Controllers/DataContracts/Reaction.cs
namespace maxhanna.Server.Controllers.DataContracts
{
    public class Reaction
    { 
        public int Id { get; set; }
        public User? User { get; set; }
        public int? CommentId { get; set; }
        public int? StoryId { get; set; }
        public int? FileId { get; set; }
        public int? MessageId { get; set; }
        public DateTime? Timestamp { get; set; }
        public string? Type { get; set; }
    }
}

File: maxhanna.client/src/services/datacontracts/reactions/reaction.ts
import { User } from "../user/user";

export class Reaction {  
  id?: number; 
  commentId?: number;
  storyId?: number;
  messageId?: number;
  fileId?: number;
  userProfileId?: number;
  user?: User;
  type?: string; 
  timestamp?: Date; 
}
